deployment/k8s-config/kustomize/base/skaha-workload/config/build-menu.sh

```diff
@@ -2,29 +2,52 @@
 
 HOST=$1
 STARTUP_DIR="/desktopstartup"
-EXECUTABLE_DIR="$HOME/.local/bin"
-DESKTOP_DIR="$HOME/.local/share/applications"
-DIRECTORIES_DIR="$HOME/.local/share/desktop-directories"
+OLD_EXECUTABLE_DIR="$HOME/.local/bin"
+OLD_DESKTOP_DIR="$HOME/.local/share/applications"
+OLD_DIRECTORIES_DIR="$HOME/.local/share/desktop-directories"
+SKAHA_DIR="$HOME/.local/skaha"
+EXECUTABLE_DIR="$HOME/.local/skaha/bin"
+DESKTOP_DIR="$HOME/.local/skaha/share/applications"
+DIRECTORIES_DIR="$HOME/.local/skaha/share/desktop-directories"
 START_ASTROSOFTWARE_MENU="${STARTUP_DIR}/astrosoftware-top.menu"
 END_ASTROSOFTWARE_MENU="${STARTUP_DIR}/astrosoftware-bottom.menu"
 MERGED_DIR="/etc/xdg/menus/applications-merged"
 ASTROSOFTWARE_MENU="${MERGED_DIR}/astrosoftware.menu"
+TERMINAL_VERSION="terminal:"
 
 init_dir () {
   if [[ -d "$1" ]]; then
+    # empty the directory
     rm -f $1/*
   else
     mkdir -p "$1"
   fi
 }
 
-init () {
+init_dirs () {
   dirs="${EXECUTABLE_DIR} ${DESKTOP_DIR} ${DIRECTORIES_DIR}"
   for dir in ${dirs}; do
     init_dir ${dir}
   done
 }
 
+delete_old_dirs () {
+  dirs="${OLD_EXECUTABLE_DIR} ${OLD_DESKTOP_DIR} ${OLD_DIRECTORIES_DIR}"
+  for dir in ${dirs}; do
+    if [[ -d "$dir" ]]; then
+      # delete the directory
+      rm -rf $dir
+    fi
+  done
+}
+
+init () {
+  if [[ ! -d "${SKAHA_DIR}" ]]; then
+    delete_old_dirs
```

Comment:
I wouldn't worry about automatic cleanup.  I means we would have to have a second release with this code removed in the future.  I think it's harmless to leave the old dirs there.

Reply:
Yes, we'll have a second release that removes the codes that handle the old dirs.

Reply:
Okay, but when would this second release occur?  Would we have to check user's home directories and see if it's been done?  What about simply removing those old scripts now manually?  I might be a good time since all sessions were stopped yesterday because of the upgrade.

Reply:
Okay, perfect, let's move towards release then.  I guess you can remove the cleanup from the scripts.

Reply:
So I'll remove the code to delete the old dirs from build-menu.sh. We'll need to run a separate script to remove the old dirs. We'll release this fix before the users start up a new session.

Reply:
Do we have time for the above?

Reply:
Sure, lots of time. If we have to skip a few users that's okay.